test/components/AuthForm_test.tsx

```diff
@@ -77,9 +78,7 @@ describe("AuthForm component", () => {
         fireEvent.change(screen.queryByLabelText("Server*"), {
           target: { value: "http://test.server/v1" },
         });
-        await screen.findByTestId("spinner"); // spinner should show up
         await waitFor(() => new Promise(resolve => setTimeout(resolve, 500))); // debounce wait
-        expect(screen.queryByTestId("spinner")).toBeNull(); // spinner should be gone by now
```

Comment:
Don't we still want to make some assertions about a spinner appearing then disappearing?